libs/l10n/l10n.ts

```diff
@@ -0,0 +1,30 @@
+type Locale =
+  | "en-US"
+  | "es"
+  | "fr"
+  | "ja"
+  | "ko"
+  | "pt-BR"
+  | "ru"
+  | "zh-CN"
+  | "zh-TW";
+type TranslatedLocale = Exclude<Locale, "en-US">;
+type Strings = {
+  "en-US": string;
+} & Record<TranslatedLocale, string>;
```

Comment:
I guess you want something like this instead, to allow the TranslatedLocale keys to be optional (otherwise you might as well just do `type Strings = Record<Locale, string>`.

I think something like LocaleString is a better type name too:

```suggestion
type LocaleString = {
  "en-US": string;
} & {
  [K in TranslatedLocale]?: string;
};
```

Reply:
Fixed in [580c0a0](https://github.com/mdn/yari/pull/11011/commits/580c0a05ba52b392b7d8c97c0c358bff6e12dc61).

(Actually, I think it's good to make all locales required.)